File: my-folder/0001-two-sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        arr = []
        for x in range(len(nums)):
            for y in range(x+1, len(nums)):
                if nums[y] == target - nums[x]:
                    arr = [x, y]
        if(len(arr) > 0):
            return arr
        else:
            return "No two sum solution"

File: my-folder/0001-two-sum/solution.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
  let index = [];
  for (let i = 0; i < nums.length; i++) {
    let sum = 0;
    for (let j = i + 1; j < nums.length; j++) {
      sum = nums[i] + nums[j];
      if (sum === target) {
        index.push(i);
        index.push(j);
        break;
      }
    }
    if (sum === target) break;
  }
  return index;
};
